src/domain/profile/state/ProfileReducers.tsx

```diff
@@ -0,0 +1,14 @@
+import { createReducer } from 'redux-starter-kit';
+
+//import { string } from 'prop-types';
+import ProfileValues from '../types/ProfileTypes';
+import { PROFILE_ACTIONS } from '../constants/ProfileActionConstants';
+
+export const defaultProfileValules: ProfileValues = {
+  firstName: '',
```

Comment:
this have to struct better, like a real user profile, can leave like this with a todo for future?